src/main/kotlin/com/colivery/serviceaping/persistence/repository/HelpRequestRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.colivery.serviceaping.persistence.repository
+
+import com.colivery.serviceaping.persistence.entity.HelpRequestEntity
+import org.springframework.data.repository.CrudRepository
+import java.util.*
+
+interface HelpRequestRepository : CrudRepository<HelpRequestEntity, UUID> {
+
+    fun findAllByAdminUser_Id(adminUserId: String): Set<HelpRequestEntity>
```

Comment:
I am not really a fan of underscores in method names. Is that necessary? I think, Spring also generates the implementation correctly without it. Also: could we use a User Entity as the argument instead of a plain string? Or at least an UUID object? Would make it a bit more typesafe :)